src/Microsoft.DotNet.XHarness.iOS/AppRunner.cs

```diff
@@ -131,7 +128,7 @@ public class AppRunner
             args.Add(new SetAppArgumentArgument($"-hostport:{listener.Port}", true));
             args.Add(new SetEnvVariableArgument("NUNIT_HOSTPORT", listener.Port));
 
-            if (_useTcpTunnel && !isSimulator) // simulators do not support tunnels
+            if (_listenerFactory.UseTunnel && !isSimulator) // simulators do not support tunnels
```

Comment:
I think it's somehow a bit weird we are asking Factory for things. Maybe this could be another return value from `Create`?

Reply:
not a good idea for now, I'll merge will create a pr that fixes things and we can take a second look.